lineage/CellVar.py

```diff
@@ -150,14 +152,14 @@ def get_subtrees(node, lineage):
     return subtree, not_subtree
 
 
-def find_two_subtrees(cell, lineage):
+def find_two_subtrees(Cell: cell, lineage: list):
```

Comment:
I don't like having a type and variable of essentially the same name. Why not call the type `cellType`, and the argument `cell` like before?